library/src-2.x/scala/internal/WithBounds.scala

```diff
@@ -0,0 +1,8 @@
+package scala.annotation.internal
+
+import scala.annotation.Annotation
+
+/** An annotation to indicate a pair of type bounds that comes with a type.
+ *  Used to indicate optional bounds of an opaque type
+ */
+class WithBounds[Lo /*<: AnyKind*/, Hi /*<: AnyKind*/] extends Annotation
```

Comment:
This file shouldn't be necessary to compile with Scala 2

Reply:
Removed it in the last commit